src/keycloak/keycloak_admin.py

```diff
@@ -3744,6 +3768,37 @@ def delete_authentication_flow_execution(self, execution_id: str) -> bytes:
             expected_codes=[HTTP_NO_CONTENT],
         )
 
+    def change_execution_priority(self, execution_id: str, diff: int) -> None:
+        """
+        Raise or lower execution priority of diff time.
+
+        :param execution_id: id of execution to lower priority
+        :type execution_id: str
+        :param diff: Integer number, raise of diff time if positive lower of diff time if negative
+        :type diff: int
+        :raises KeycloakPostError: when post requests are failed
+        """
+        params_path = {"id": execution_id, "realm-name": self.connection.realm_name}
+        try:
+            if diff > 0:
+                for _i in range(diff):
+                    _ = self.connection.raw_post(
```

Comment:
the `raw_post` method does not throw if the request returns 400+.  These calls should be wrapped with `raise_error_from_response` function. This should be sufficient

```
for _ in range(diff):
    resp = self.connection.raw_post(...)
    raise_error_from_response(resp, KeycloakPostError)
```